grpc/common/common.go

```diff
@@ -127,12 +127,20 @@ func (m *metric) Unit() string                  { return m.unit }
 
 // Convert common.Metric to core.Metric
 func ToCoreMetric(mt *Metric) core.Metric {
+	var lastAdvertisedTime time.Time
+	// if the lastAdvertisedTime is not set we handle.  -62135596800 represents the
+	// number of seconds from 0001-1970 and is the default value for time.Unix.
+	if mt.LastAdvertisedTime.Sec == int64(-62135596800) {
```

Comment:
This is due to timezone switch to LMT (not sure the exact cut off dates). Maybe another option is to only log in UTC?

```
$ go_time
t: 1900-01-01 00:00:00 +0000 UTC t in Local: 1899-12-31 16:07:02 -0752 LMT
t2: 2000-01-01 00:00:00 +0000 UTC t2 in Local: 1999-12-31 16:00:00 -0800 PST
```